diff -Nur liblzf-0.4.orig/Makefile.am liblzf-0.4/Makefile.am
--- liblzf-0.4.orig/Makefile.am	Thu Jan  1 01:00:00 1970
+++ liblzf-0.4/Makefile.am	Sat Jul 27 10:44:46 2002
@@ -0,0 +1,9 @@
+lib_LTLIBRARIES = liblzf.la
+liblzf_la_SOURCES = lzf_c.c lzf_d.c
+
+include_HEADERS = lzf.h
+
+bin_PROGRAMS = lzf
+
+lzf_SOURCES = lzf.c
+lzf_LDADD = liblzf.la
--- liblzf-2.0/configure.ac.orig	2006-10-12 15:51:11.000000000 +0200
+++ liblzf-2.0/configure.ac	2007-02-19 20:55:37.526117217 +0100
@@ -1,4 +1,5 @@
-AC_INIT
+AC_INIT(liblzf, 2.0)
+AM_INIT_AUTOMAKE
 AC_CONFIG_SRCDIR([lzfP.h])
 
 AC_CONFIG_HEADER(config.h)
@@ -6,7 +7,7 @@
 AC_GNU_SOURCE
 AC_SYS_LARGEFILE
 AC_PROG_CC
-AC_PROG_RANLIB
+AC_PROG_LIBTOOL
 AC_PROG_INSTALL
 AC_HEADER_STDC
 
@@ -19,11 +20,5 @@
 AC_CHECK_HEADERS(getopt.h)
 AC_CHECK_FUNCS(getopt_long)
 
-if test "$GCC" = yes; then
-   CFLAGS="$CFLAGS -O3 -funroll-all-loops"
-else
-   AC_MSG_RESULT(no gcc)
-fi
-
 AC_CONFIG_FILES([Makefile])
 AC_OUTPUT
